filters/addbackground.go

```diff
@@ -58,12 +70,11 @@ func (r *addBackground) CreateFilter(args []interface{}) (filters.Filter, error)
 		return nil, err
 	}
 
-
 	return f, nil
 }
 
-func (r *addBackground) Request(ctx filters.FilterContext) {}
+func (s *addBackground) Request(ctx filters.FilterContext) {}
 
-func (r *addBackground) Response(ctx filters.FilterContext) {
-	HandleImageResponse(ctx, r)
+func (s *addBackground) Response(ctx filters.FilterContext) {
+	HandleImageResponse(ctx, s)
```

Comment:
Question: it is not completely clear to me why the HandleImageResponse is called only from the addBackground filter